docker-deploy.gemspec

```diff
@@ -2,16 +2,16 @@ $LOAD_PATH.push File.expand_path('../lib', __FILE__)
 
 Gem::Specification.new do |s|
   s.name        = 'docker-deploy'
-  s.version     = '1.1.0'
+  s.version     = '1.1.1'
   s.date        = '2014-11-19'
   s.summary     = 'Docker deployment utilities for AWS Elastic Beanstalk'
   s.description = ''
-  s.authors     = ['ad2games GmbH']
-  s.email       = 'developers@ad2games.com'
+  s.authors     = ['Combostrike GmbH']
+  s.email       = 'developers@combostrike.com'
   s.files       = Dir['lib/**/*']
-  s.homepage    = 'http://www.ad2games.com'
+  s.homepage    = 'http://www.combostrike.com'
   s.license     = ''
 
   s.add_dependency 'rake'
-  s.add_dependency 'bundler-audit'
+  s.add_dependency 'bundler-audit', '~> 0.8.0'
```

Comment:
Why do you need to restrict the version?

Reply:
I don't want to restrict it. I want to update. But I feel it is always better to have something specific to avoid unexpected behaviour

Reply:
Unless there is a specific incompatibility you know about, I'd leave it as open as possible.

Reply:
Okay

Reply:
At least that's how we handled it so far. Restricting the major version _would_ make sense, though... :thinking: That would be `'~> 0.8'`, then.